app/views/newflow/login_signup/educator_signup_form.html.erb

```diff
@@ -63,25 +62,38 @@
         <div class="content control-group">
           <%= label_tag :phone_number, I18n.t(:"login_signup_form.phone_number_label"), class: 'field-label' %>
           <%=
-            f.text_field :phone_number,
-                         type: 'tel',
-                         class: 'int-country-code form-control wide',
-                         placeholder:  I18n.t(:'login_signup_form.phone_number_placeholder'),
-                         onkeydown: 'javascript:backspacerDOWN(this,event);',
-                         onkeyup: 'javascript:backspacerUP(this,event);'
+            fh.text_field name: :phone_number,
+                          type: 'tel',
+                          supplemental_class: 'int-country-code form-control wide',
+                           placeholder:  I18n.t(:'login_signup_form.phone_number_placeholder'),
+                           onkeydown: 'javascript:backspacerDOWN(this,event);',
+                           onkeyup: 'javascript:backspacerUP(this,event);'
 
           %>
         </div>
 
-        <div class="content control-group">
+        <div class="content control-group email-input-group newflow">
           <%= label_tag :email, I18n.t(:"login_signup_form.school_issued_email_label"), class: 'field-label' %>
           <%=
             fh.text_field(
               name: :email,
+              supplemental_class: 'signup_email',
```

Comment:
It would **super** if you could rename this class name to something else because there is already an id called `signup_email`, to prevent confusion in the future.